### PR TITLE
Added doc for email support

### PR DESCRIPTION
**Pull request by nikhilgupta1211**
To be rebased after #76

### CODE REVIEW

File: doc/EMAIL.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Sending Emails in Travel Support Program
+
+To send emails with travel support program you need to setup some external mail service. For example using the gmail smtp, just add the smtp settings using `config.action_mailer.smtp_settings` in the `environment` files.
+For gmail smtp settings you can checkout this <a href="https://support.google.com/a/answer/176600?hl=en">link</a>.
+
+There are various options provided for emails like `async_emails`, `email_from`, `email_footer`, `email_default_url_options` which can be set in the site.yml file.
```
Comment (bgeuken) [inline doc/EMAIL.md]: If this hasn't been documented yet, it would be good to do it now. Could you do that?

Comment (nikhilgupta1211) [inline doc/EMAIL.md]: Ok :)

File: doc/INSTALL.MD [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+To send emails with travel support program you need to setup some external mail service. For example using the gmail smtp, just add the smtp settings using `config.action_mailer.smtp_settings` in the `environment` files.
+For gmail smtp settings you can checkout this <a href="https://support.google.com/a/answer/176600?hl=en">link</a>.
+
+Their are some email variables provided in the site.yml file which your can use to configure emails in TSP :
+
+| Variable          | Content           | Purpose               |
+|----------         |---------          |---------                  |
+| async_emails      | true              | To use delayed_jobs for emails         |
+| email_from         | Travel Support Program <noreply@example.com>       | Address used by TSP to send emails  |
```
Comment (bgeuken) [inline doc/INSTALL.MD]: Minor thing, but please align the second | with the one above

File: doc/INSTALL.MD [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+## Sending Emails in Travel Support Program
+
+To send emails with travel support program you need to setup some external mail service. For example using the gmail smtp, just add the smtp settings using `config.action_mailer.smtp_settings` in the `environment` files.
+For gmail smtp settings you can checkout this <a href="https://support.google.com/a/answer/176600?hl=en">link</a>.
+
+Their are some email variables provided in the site.yml file which your can use to configure emails in TSP :
+
+| Variable          | Content           | Purpose               |
+|----------         |---------          |---------                  |
+| async_emails      | true              | To use delayed_jobs for emails         |
```
Comment (bgeuken) [inline doc/INSTALL.MD]: Please either align all of the 4th | chars or remove the whitespaces before 'To use delayed_jobs for emails'

Comment (bgeuken) [inline doc/INSTALL.MD]: True would mean the mails get send directly, right? I would rephrase the purpose. It should become clear what both options do.

Comment (nikhilgupta1211) [inline doc/INSTALL.MD]: yes ok

Comment (nikhilgupta1211) [inline doc/INSTALL.MD]: true means mails will be send using delayed_jobs

File: doc/INSTALL.MD [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+| Variable          | Content                                       | Purpose                              |
+|----------         |---------                                      |---------                             |
+| async_emails      | true/false                                    | Enable the usage of delayed_jobs     |
+| email_from        | Travel Support Program <noreply@example.com>  | Address used by TSP to send emails   |
+| email_footer      | Travel Support Program                        | Footer for email                     |
+ 
+### Testing emails
+
+For testing emails `letter_opener_web_gem` is provided in TSP. To use it just set the `config.action_mailer.delivery_method` to `:letter_opener` . If you are running your app from a Vagrant machine or want to test the emails in production you can set the delivery_method option to `:letter_opener_web`.
```
Comment (ChrisBr) [inline doc/INSTALL.MD]: Please use a link to the gem instead of just marking it as ``code``

Comment (ChrisBr) [inline doc/INSTALL.MD]: You should also mention how to use ``letter_opener_web`` (-> url)

Comment (nikhilgupta1211) [inline doc/INSTALL.MD]: ok :)
